app/build.gradle

```diff
@@ -3,9 +3,9 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
 
-def keystorePropertiesFile = rootProject.file("keystore.properties")
```

Comment:
this is so wrong, you should not change anything in this file. Just comment it out locally if needed.

Reply:
Not able to run the project without commenting  :/

Reply:
you can comment it on your pc, just dont submit the changes to the pull request